<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="27.5740" tests="2" failures="1">
  <testsuite name="Root Suite" timestamp="2021-09-12T11:39:25" tests="0" file="cypress\integration\calculateBorrowingPower.feature" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Calculate Borrowing Power" timestamp="2021-09-12T11:39:25" tests="2" time="27.5740" failures="1">
    <testcase name="Calculate Borrowing Power Calculate Borrowing Power (example #1)" time="5.4070" classname="Calculate Borrowing Power (example #1)">
      <failure message="`cy.visit()` failed trying to load:

https://www.anz.com.au/personal/home-loans/calculators-tools/much-borrow/

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  &gt; Error: getaddrinfo ENOTFOUND www.anz.com.au

Common situations why this would fail:
  - you don&apos;t have internet access
  - you forgot to run / boot your web server
  - your web server isn&apos;t accessible
  - you have weird network configuration settings on your computer" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

https://www.anz.com.au/personal/home-loans/calculators-tools/much-borrow/

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: getaddrinfo ENOTFOUND www.anz.com.au

Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer
    at http://localhost:54813/__cypress/runner/cypress_runner.js:159135:23
    at visitFailedByErr (http://localhost:54813/__cypress/runner/cypress_runner.js:158495:12)
    at http://localhost:54813/__cypress/runner/cypress_runner.js:159134:11
    at tryCatcher (http://localhost:54813/__cypress/runner/cypress_runner.js:10765:23)
    at Promise._settlePromiseFromHandler (http://localhost:54813/__cypress/runner/cypress_runner.js:8700:31)
    at Promise._settlePromise (http://localhost:54813/__cypress/runner/cypress_runner.js:8757:18)
    at Promise._settlePromise0 (http://localhost:54813/__cypress/runner/cypress_runner.js:8802:10)
    at Promise._settlePromises (http://localhost:54813/__cypress/runner/cypress_runner.js:8878:18)
    at _drainQueueStep (http://localhost:54813/__cypress/runner/cypress_runner.js:5472:12)
    at _drainQueue (http://localhost:54813/__cypress/runner/cypress_runner.js:5465:9)
    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:54813/__cypress/runner/cypress_runner.js:5481:5)
    at Async.drainQueues (http://localhost:54813/__cypress/runner/cypress_runner.js:5351:14)
From Your Spec Code:
    at Function.Visit (http://localhost:54813/__cypress/tests?p=cypress\integration\calculateBorrowingPower.feature:77:10)
    at Context.eval (http://localhost:54813/__cypress/tests?p=cypress\integration\calculateBorrowingPower.feature:208:37)
    at Context.resolveAndRunStepDefinition (http://localhost:54813/__cypress/tests?p=cypress\integration\calculateBorrowingPower.feature:11608:29)
    at Context.eval (http://localhost:54813/__cypress/tests?p=cypress\integration\calculateBorrowingPower.feature:10929:35)

From Node.js Internals:
  Error: getaddrinfo ENOTFOUND www.anz.com.au
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:66:26)
  ]]></failure>
    </testcase>
    <testcase name="Calculate Borrowing Power Validate form is clear after clickiing on start over button (example #1)" time="22.1670" classname="Validate form is clear after clickiing on start over button (example #1)">
    </testcase>
  </testsuite>
</testsuites>